roles/uperf/templates/workload.yml.j2

```diff
@@ -88,13 +88,15 @@ spec:
             export h={{item.spec.clusterIP}};
 {% else %}
 {% if workload_args.multus.client is defined %}
+          - export multus_client={{workload_args.multus.client}}
```

Comment:
I think this is missing "" 's around it. Or at least the opening one and remove the opener from the below line.